cfn-resources/scripts/update-sdk-preview.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+echo "Script will update to SDL Dev Preview. Note: you need to have the latest major version in order to be able to use it."
```

Comment:
Script started as helper for cli. No we have it in 3 different repos (still ok). The more we grow we should consider creating github action with those scripts or have org level workflows

Reply:
yeah, an APIx common repo would be great